testing/tests/Memory Leaks/elementsOnDispose_bundled.tests.js

```diff
@@ -34,6 +34,7 @@ $.each(DevExpress.ui, function(componentName) {
                 $(testNode)[componentName](memoryLeaksHelper.getComponentOptions(componentName))[componentName]('instance');
                 this.clock.tick(0);
                 memoryLeaksHelper.destroyTestNode(testNode);
+                this.clock.tick(100);
```

Comment:
What kind of timeouts this tick covers?

Reply:
There is a timeout in preact/hooks. It will be cleared after 100ms after destroying a node.

Reply:
https://github.com/preactjs/preact/blob/master/hooks/src/index.js#L249-L261